phoenicis-wine/src/main/java/com/playonlinux/framework/Wine.java

```diff
@@ -69,12 +70,46 @@
 @ScriptClass
 public class Wine implements SetupWizardComponent {
     private static final Logger LOGGER = Logger.getLogger(Wine.class);
-    private static final Architecture DEFAULT_ARCHITECTURE = Architecture.I386;
+
+    private static final String EXCEPTION_PREFIX_INCOMPATIBLE_ARCH = "A 32bit wineprefix cannot execute 64bits executables!"; //$NON-NLS-1$
```

Comment:
I'm not a fan of using constant for log strings. It makes the code really hard to read.